lab_1/main.py

```diff
@@ -53,10 +75,26 @@ def get_top_n_words(freq_dict: dict, top_n: int) -> list:
     top_n = 1
     --> ['happy']
     """
+    list_of_values = []
+    for value in freq_dict.values():
+        if value not in list_of_values:
+            list_of_values += [value]
+    list_of_values = sorted(list_of_values, reverse=True)
+    top_words = []
+
+    for freq in list_of_values:
+        common_freq_words = []
+
+        for keys in freq_dict:
+            if freq_dict[keys] == freq:
```

Comment:
suppose that in list_of_values there are repetitions, like 9 9 9, so  there are several words with the same frequency
they all be extracted using this line

and they will be extracted twice more when iterating over the list on line 85